content/id/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -0,0 +1,133 @@
+
+Terkadang, Kubernetes menentukan nilai dari `ownerReference` secara otomatis. Sebagai contoh, ketika kamu membuat sebuah ReplicaSet, Kubernetes secara otomatis akan menentukan tiap kolom `ownerReference` dari tiap Pod di dalam ReplicaSet. Pada versi 1.8, Kubernetes secara otomatis menentukan nilai dari `ownerReference` untuk objek yang diciptakan atau diadopsi oleh ReplicationController, ReplicaSet, StatefulSet, DaemonSet, Deployment, Job dan CronJob.
+
+Kamu juga bisa menspesifikasikan hubungan antara pemilik dan _dependent_ dengan cara menentukan kolom `ownerReference` secara manual.
+
+Berikut adalah berkas untuk sebuah ReplicaSet yang memiliki tiga Pod:
+
+{{< codenew file="controllers/replicaset.yaml" >}}
+
+Jika anda membuat ReplicaSet tersebut dan kemudian melihat metadata Pod, kamu akan melihat kolom OwnerReferences:
+
+```shell
+kubectl apply -f https://k8s.io/examples/controllers/replicaset.yaml
+kubectl get pods --output=yaml
+```
+
+Keluaran menunjukkan bahwa pemilik Pod adalah sebuah ReplicaSet bernama `my-repset`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  ...
+  ownerReferences:
+  - apiVersion: apps/v1
+    controller: true
+    blockOwnerDeletion: true
+    kind: ReplicaSet
+    name: my-repset
+    uid: d9607e19-f88f-11e6-a518-42010a800195
+  ...
+```
+{{< note >}}
+Referensi pemilik lintas _namespace_ tidak diperbolehkan oleh desain. Artinya:
+1) _Namespace-scoped dependent_ hanya bisa menspesifikasikan pemilik jika berada di _namespace_ yang sama, dan pemilik bersifat _cluster-scoped_. 
+2) _Cluster-scoped dependent_ hanya bisa menspesifikasikan pemilik bersifat _cluster-scoped_, tetapi tidak berlaku untuk pemilik bersifat _namespace-scoped_.
+{{< /note >}}
+
+## Mengontrol bagaimana _garbage collector_ menghapus _dependent_
+
+Ketika kamu menghapus sebuah objek, kamu bisa menspesifikasi apakah _dependent_ objek tersebut juga dihapus secara otomatis. Menghapus _dependent_ secara otomatis disebut _cascading deletion_. _Cascading deletion_ memiliki dua mode: _background_ dan _foreground_
+
+### Foreground cascading deletion
+
+Pada *foreground cascading deletion*, pertama objek utama akan memasuki keadaan "_deletion in progress_". Pada saat keadaan "_deletion in progress_", kondisi-kondisi berikut bernilai benar:
+
+ * Objek masih terlihat via REST API
+ * `deletionTimestamp` objek telah ditentukan
+ * `metadata.finalizers` objek memiliki nilai `foregroundDeletion`.
+
+ Ketika dalam keadaan "_deletion in progress_", _garbage collector_ menghapus _dependent_ dari objek. Ketika _garbage collector_ telah menghapus semua "_blocking_" _dependent_ (objek dengan `ownerReference.blockOwnerDeleteion=true`), _garbage collector_ menghapus objek pemilik.
```

Comment:
nit: please change all _dependent_ to dependen

Reply:
fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305